docs/modules/computing/pages/entry-processor.adoc

```diff
@@ -95,6 +95,25 @@ You may also return `null` from the `EntryProcessor#getBackupProcessor` method.
 
 NOTE: It is possible that an entry processor could see that a key exists though its backup processor may not find it due to an unsent backup of a previous operation, e.g., a previous put operation. In those situations, Hazelcast internally/eventually synchronizes those owner and backup partitions so you do not lose any data. When coding a backup entry processor, you should take that case into account, otherwise `NullPointerException` can be seen since `Map.Entry.getValue()` may return `null`.
 
+=== Updating Entry TTLs
+
+You can update the time-to-live (TTL) duration and value of existing map entries in your entry processor application, using the `ExtendedMapEntry` interface.
+The following is an example snippet where the entries are incremented by one and provided with a new TTL.
+
+[source,java]
+----
+ExtendedMapEntry<Integer, Integer> entry = (ExtendedMapEntry<Integer, Integer>) e;
+int newValue = entry.getValue() + 1;
+if (ttlSeconds > 0) {
```

Comment:
since there is no `ttlSeconds` field mentioned anywhere before we can:
- either present the full class definition from `EntryProcessor`'s [javadoc](https://github.com/hazelcast/hazelcast/blob/7c3894b1474dba56648cf10ad8441615486fcdea/hazelcast/src/main/java/com/hazelcast/map/EntryProcessor.java#L66-L88) 
- or skip the `ttlSeconds` condition and just demonstrate set-with-ttl variant. 
```
...
int newValue = entry.getValue() + 1;
// set TTL to 10 seconds
entry.setValue(newValue, 10, TimeUnit.SECONDS); 
```